### DISCUSSION TITLE
Section Numbering

### DISCUSSION THREAD

**Discussion by tknopp** [fonsp/Pluto.jl#1615] [Questions about using Pluto]
Hi, I would be interested if there is a possibility to get automatic section numbering. I now do it manually, which is of course not a nice solution. I need this for creating lecture material.

**Comment by disberd** [comment]
You can find some discussion here: https://discourse.julialang.org/t/pluto-is-possible-to-auto-numbering-for-headings/66504

But no clear and ready solution there.
